CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_mixpanel v0.7.1
```

Comment:
I think this should be a breaking change. Not that this is breaking so to speak, but Databricks and Postgres compatibility deserve their own breaking release imo.

Reply:
Can you make the version bump change throughout the rest of the package as well.